### ISSUE TITLE
Docs: Don't load jQuery from 3rd party CDN for v4 Docs

### ISSUE THREAD

**Issue by coliff** [twbs/bootstrap#32668]
Suggestion: on the v4 docs jQuery is loaded from the jQuery CDN (https://code.jquery.com/jquery-3.5.1.slim.min.js), with a local fallback. It'd be less HTML code and likely always better for performance if it always loaded jQuery locally. It'll save the cost of opening a new TCP connection (DNS resolution, TCP handshake and TLS negotiation). Browsers don't have Cross-Domain caching so there's no advantage in using a 3rd party CDN anymore (as your site is served from a CDN anyway).
Good explainer here (with Bootstrap being used as the example):
https://csswizardry.com/2019/05/self-host-your-static-assets/

**Comment by XhmikosR** [comment]
Yeah, it's low priority for sure since we have moved to v5, but I guess we could look into it for the next 4.x version.

**Comment by XhmikosR** [comment]
So, I don't think I'd like to make this change right now, and add back a 3rd-party file to track and update. If/when we manage to handle all of our 3rd-party assets through npm, we could definitely do it.

What we could do right now is switch to jsDelivr thus one less CDN.

**Comment by coliff** [comment]
sounds good. 👍

**Comment by XhmikosR** [comment]
@coliff I'm going to close this for now. We have switched to using jsDelivr for all assets so this should still be better.
